README.md

```diff
@@ -24,6 +24,10 @@ $ flatpak install flatseal.flatpak
 
 Or just use [Builder](https://flathub.org/apps/details/org.gnome.Builder)
 
+## Documentation
+
+See [DOCUMENTATION.md](./DOCUMENTATION.md).
```

Comment:
Maybe something like:

For a detailed description of supported permissions, see the [documentation](./DOCUMENTATION.md) page.

?

Reply:
Done https://github.com/tchx84/Flatseal/commit/9a4e417d3a3548a4406cfcb88fd4b97dd1c31dcf